### DISCUSSION TITLE
Looking for a concise way to list fields and levels

### DISCUSSION THREAD

**Discussion by J-J-Abram** [dtcenter/METplus#1513] [Configuration] [answered]
I'm currently specifying multiple fields and pressure levels within a Gridstat METplus config file (INI format).

In order to maintain that the levels within the netcdf output files are consistent with the pressure levels of the fields, I've specified set_attr_level for each pressure level for each field (see attached config file lines 82-322 - converted to a txt file in order to paste in this discussion.). We want to ensure that the pressure levels are the same for each field in the netcdf output to allow easy comparison to our current verification software. 

This generates multiple variables and very long config files which become unwieldy quickly as the number of pressure levels requested increases for multiple fields. I know that pressure levels can be requested in list form to keep the config file shorter, but can the set_attr_levels be specified in a similar list format? (See lines 325-332 for some list examples)

Any suggestions on how to reduce the length of the config file would be greatly appreciated.

Thanks

[Gridstat_METplus_Global.txt](https://github.com/dtcenter/METplus/files/8253721/Gridstat_METplus_Global.txt)

**Comment by CPKalb** [comment, marked as answer]
Hi:

Currently there is not a way to simplify the logic for set_attr_levels.  However, there are 2 open issues that may help, and would address this in  the future.

https://github.com/dtcenter/MET/issues/2101
https://github.com/dtcenter/MET/issues/1815

Christina

**Comment by CPKalb** [comment]
Hi:

I wanted to check in and see if you have additional questions on this.  If not, I'd like to mark this as answered.

Thanks,
Christina

**Comment by J-J-Abram** [comment]
Hi Christina,

Thanks very much. Happy for you to mark this as answered.

Cheers

Joe
